src/prelexer.cpp

```diff
@@ -1578,6 +1589,50 @@ namespace Sass {
                       >(src);
     }
 
+    extern const char css_variable_url_negates[] = "()[]{}\"'#/";
+    const char* css_variable_value(const char* src) {
+      return sequence<
+        alternatives<
+          sequence<
+            negate< sequence<
+              exactly<'u'>,
+              exactly<'r'>,
+              exactly<'l'>,
```

Comment:
Use `url_kwd` or maybe introduce a `urlfn_kwd` for reuse (seems we need both).

Reply:
Updated